src/App.tsx

```diff
@@ -1,18 +1,42 @@
 import React from 'react';
+import { tabs } from './api/data';
 
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
-const tabs: Tab[] = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-];
+type State = {
+  selectedTab: string,
+};
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedTab: 'tab-1',
+  };
 
-export default App;
+  onTabSelected = (selectedTab: Tab) => {
+    this.setState({
+      selectedTab: tabs.find(
+        tab => tab.id === selectedTab.id,
+      )?.id || 'tab-1',
+    });
+  };
+
+  render() {
+    const { selectedTab } = this.state;
+
+    return (
+      <div className="App">
+        <h1>{`Selected tab is ${tabs.find(tab => selectedTab === tab.id)?.title}`}</h1>
+
+        <Tabs
+          tabs={tabs}
+          onTabSelected={this.onTabSelected}
+        />
+
+        <p>
+          {tabs.find(tab => selectedTab === tab.id)?.content}
```

Comment:
You can make variable for `tabs.find(tab => selectedTab === tab.id)` so you don't have to do it twice